run/run.py

```diff
@@ -345,6 +358,28 @@ def get_config():
     return config['default']
 
 
+def patch_wpt(config, platform):
+    '''Applies util/wpt.patch to WPT.
+
+    The patch is necessary to keep WPT running on long runs.
+    jeffcarp has a PR out with this patch:
+    https://github.com/w3c/web-platform-tests/pull/5774
+    '''
+    patch_path = '%s/util/wpt.patch' % config['wptd_path']
+    with open(patch_path) as f:
+        patch = f.read()
+
+    # Hackery since the --sauce-platform command line arg doesn't
```

Comment:
Nit: You could perhaps remove "Hackery since". The fact that you have to explain it in a comment implies that there's some hackiness :-P